### ISSUE TITLE
Data Extractor Output Missing Data

### ISSUE THREAD

**Issue by GTI-Cody** [OpenC3/cosmos#386]
**Describe the bug**
When using the data extractor on a large number of packets, ~180 in our case, the output file is mostly empty. The only data available are random columns that contain time stamps and 0. The column does not line up with the packet in that row. If I look at the row for packet A,B,C or D I get a nonsense time stamp in a column that corresponds to packet X. There is no usable data present in these files. This was only 15 minutes of data. It breaks somewhere between 50 packets and ~180. I didn't push past 50 due to time constraints. 

**To Reproduce**

Try to use data extractor on a large number of packets.  

**Expected behavior**
For there to be data corresponding to the packets I chose to extract data for. 

**Environment (please complete the following information):**

- OS: Windows 10
- OpenC3 COSMOS Version 5.3.0
- Browser
  -- [] Internet Explorer (IE)
  -- [X] Google Chrome
  -- [] Chromium based browsers
  -- [] Mozilla Firefox
  -- [] Safari

**Comment by ryanmelt** [comment]
I've seen that DataExtractor doesn't handle binary data well.  If you add the IMAGE packet from our demo it adds lots of blank lines.  That might be the cause here too.

**Comment by GTI-Cody** [comment]
It isn't just blank lines. Most (~99%) columns are completely empty. Will report back when we get a chance to run the demo. Sorry about not including that in the main post.

**Comment by ryanmelt** [comment]
With exporting lots of packets most of the columns will be empty.  
Every line is a single packet, and it only includes the data (columns) for that packet (and a bunch of empty columns for the other packets).
If you enable Mode -> Fill Down, it will fill down the empty columns from previous packets.

If things get misaligned at some point, my guess that is because of binary data somewhere or a comma in data

**Comment by GTI-Cody** [comment]
I went through our packets in chunks of 20 and I was able to get a good csv (tab delimited) for each chunk. If I try the same exact time period against the entire target I get a file, as previously described, that is unusable. I get at most 1 column entry per row, and some rows with no data at all, outside of the TIME, TARGET, and PACKET columns. 

To me this shows the issue isn't with the contents of the packets (commas or binary), but with the number of packets. I'm able to parse the files with 20 packets just fine. The full target worth of packets (~180) is simply missing most of the data. I get at best 1 column per row instead of the expected 1 packet (N columns) worth of data per row. 

I also tried the full target with tab delimited, and Fill Down. This resulted in the same mostly blank file. I tried the same settings on a set of 10 packets and the file was correctly populated and filled down.
